src/dosext/mfs/mfs.c

```diff
@@ -1335,12 +1336,17 @@ static struct dir_list *get_dir(char *name, char *mname, char *mext, int drive)
   char fname[8];
   char fext[3];
 
-  if(is_dos_device(name) || !find_file(name, &sbuf, drive, NULL))
+  if (is_dos_device(name) || !find_file(name, &sbuf, drive, NULL)) {
```

Comment:
Do we need this find_file() here?

Reply:
Well I'd rather use `stat()` myself

Reply:
Its probably not needed, opendir() will later fail anyway, no?

Reply:
I expect so, but why was it there originally?

Reply:
Maybe `git blame` knows. :)